README.md

```diff
@@ -1,70 +1,127 @@
+  **-Home page :**
 
-### `npm start`
+1. When the page loads, the web app shows the list of categories that could be filtered out by some parameter, for example by the category name.
+2. Along with the category name, it will display some numeric values per category that come from the API.
+3. When the user clicks (or taps) on a category item, the application navigates to the details page.
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+**-Details page**
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+1. In the details page, the web app retrieves data from the API to show detailed data in the category.
+2. When the user clicks on the "Back" button (<), the user navigates to the home page.
 
-### `npm test`
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+## 🛠 Built With <a name="built-with"></a>
 
-### `npm run build`
+### Tech Stack <a name="tech-stack"></a>
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+    <li><a href="https://redux-toolkit.js.org/">Redux toolkit</a></li>
+  </ul>
+</details>
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+<!-- Features -->
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+### Key Features <a name="key-features"></a>
 
-### `npm run eject`
+- **Fetch Cryptocurrencies[Render Detail Pages Dynamically]**
+- **Fetch all data corresponding to a cryptocurrency from corresponding API**
+- **Filtering by different coins**
 
-**Note: this is a one-way operation. Once you `eject`, you can't go back!**
+## Overview
 
-If you aren't satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+The Loom video Link : https://www.loom.com/share/e4b38fb8217245fb94b363cb6f302a44 
 
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you're on your own.
+<!-- LIVE DEMO -->
 
-You don't have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn't feel obligated to use this feature. However we understand that this tool wouldn't be useful if you couldn't customize it when you are ready for it.
+## 🚀 Live Demo <a name="live-demo"></a>
```

Comment:
* Kindly provide a working link to your deployed project as it's a [ requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#:~:text=Deploy%20the%20project%20and%20test%20for%20final%20details.)

Reply:
I have made the proper deployment with Netlify. The link is attached here for your reference:

https://capstone-react-redux-chuaindia.netlify.app/